City-of-Pori/pori-kada
# PORI-394 Added link to news archive to search page

To test:

1. drush fra -y && drush cc all
2. Go to search page https://local.pori.fi/search
3. Make sure you can see the text under the search bar "Didn't find what you are looking for? Try searching from the news archive."
4. Go to https://local.pori.fi/en/admin/config/regional/translate/translate and search for Didn't
5. Translate the English version to Etkö löytänyt etsimääsi? Kokeile <a href="/uutisarkisto">hakea uutisarkistosta.</a>
6. Go to https://local.pori.fi/search and make the translation works. Now try the English version of search as well https://local.pori.fi/en/search and make sure it works too.
